packages/sw-background-sync-queue/demo/sw.js

```diff
@@ -16,20 +24,17 @@ let bgQueue = new goog.backgroundSyncQueue.BackgroundSyncQueue({callbacks:
 	},
 });
 
+const requestWrapper = new goog.runtimeCaching.RequestWrapper({
+  behaviors: [bgQueue],
+});
 
-self.addEventListener('fetch', function(e) {
-	if (e.request.url.startsWith('https://jsonplaceholder.typicode.com')) {
-		const clone = e.request.clone();
-		e.respondWith(fetch(e.request).catch((err)=>{
-			bgQueue.pushIntoQueue({
-				request: clone,
-			});
-			throw err;
-		}));
-	}
+const route = new goog.routing.ExpressRoute({
+  path: '/*',
+  origin: 'https://jsonplaceholder.typicode.com',
```

Comment:
As of https://github.com/GoogleChrome/sw-helpers/pull/158, you can pass in `/*`, but it won't trigger a match for cross-origin requests. If you saw your route match a cross-origin request, it might have been because you didn't have that updated code in effect.

`origin` has never been supported in `ExpressRoute`. It was just ignored.